api/v1alpha1/costmanagement_types.go

```diff
@@ -68,6 +77,24 @@ type AuthenticationSpec struct {
 	AuthenticationSecretName string `json:"secret_name,omitempty"`
 }
 
+// UploadSpec defines the desired state of Authentication object in the CostManagementSpec
+type UploadSpec struct {
+
+	// UploadWait is a field of CostManagement to represent the time to wait before sending an upload.
+	// +optional
+	// +kubebuilder:validation:Minimum=0
+	UploadWait *int64 `json:"upload_wait,omitempty"`
+
+	// UploadCycle is a field of CostManagement to represent the number of hours between each upload schedule
```

Comment:
Maybe mention the default is 6 hours? Do we want to do this in hour or minutes (default 3600) to allow us to be able to more easily test the upload cycle?

Reply:
Added.